### ISSUE TITLE
Xcolor interferes with math

### ISSUE THREAD

**Issue by vladistan** [wiki2beamer/wiki2beamer#40]
I have the following slide

```
==== Title ====

$$
    \mathbb{S} := \{ t_1, t_2, \.\.\., t_{n} \}
$$

```

when it's transformed into Latex code it becomes:

```
$$
    \mathbb{S} := \{ t\textcolor{1, t}{2, \.\.\., t}{n} \}
$$

```

which is obviously not what I expect.  Perhaps,  the color rule should not apply while in the display math mode.   Unless I am missing something I should quote my math with

**Comment by esc** [comment]
Yeah, the color rule isn't very well choosen syntax, I often have filenames with multiple underscores and it always breaks wiki2beamer...
